### ISSUE TITLE
predicted results on the test set

### ISSUE THREAD

**Issue by kosuke1701** [luka-group/lite#2]
Thanks for sharing your great work and codes!

I want to compute different metrics, e.g. MRR, of the results reported in Table 2 of the paper. Would it be possible to share us predicted labels and scores on the test set of the ultra-fine entity typing dataset?

Best regards.

**Comment by VincentLeebang** [comment]
> Thanks for sharing your great work and codes!
> 
> I want to compute different metrics, e.g. MRR, of the results reported in Table 2 of the paper. Would it be possible to share us predicted labels and scores on the test set of the ultra-fine entity typing dataset?
> 
> Best regards.

Hi, Thanks for issuing this. I'll upload the results and evaluation code later.

**Comment by VincentLeebang** [comment]
> 

Hi, you can download our pre-trained LITE checkpoint and its result file now. Check README to get the google drive link. Feel free to open another issue if you have any other question. Thx

**Comment by kosuke1701** [comment]
Thank you very much for sharing results!
